areas/appservice/src/AzureMcp.AppService/Options/AppServiceOptionDefinitions.cs

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace AzureMcp.AppService.Options;
+
+public static class AppServiceOptionDefinitions
+{
+    public const string AppName = "app-name";
+    public const string DatabaseType = "database-type";
+    public const string DatabaseServer = "database-server";
+    public const string DatabaseName = "database-name";
+    public const string ConnectionString = "connection-string";
```

Comment:
We are trying to only have -name in the options name if absolutely required.  I think we can go app and database.